src/include/kdbmount.h

```diff
@@ -0,0 +1,49 @@
+/* Mount API for newbackend, currently re-implementation of old mounting process in C instead of C++ */
+
+#ifndef KDB_MOUNT_H
+#define KDB_MOUNT_H
+
+#include <stdbool.h>
+#include <kdb.h>
+
+
+struct cBackendInfo {
```

Comment:
I don't think everything should start with c.

```suggestion
struct BackendInfo {
```